spec/module.dd

```diff
@@ -113,12 +113,12 @@ $(GNAME PackageName):
     $(GLINK_LEX Identifier)
 )
 
-$(P The $(I Identifier)s preceding the rightmost are the $(I Packages) that the
+$(P The $(I Identifier)s preceding the rightmost $(I Identifier) are the $(I Packages) that the
 module is in. The packages correspond to directory names in the source file
 path. Package and module names cannot be $(GLINK_LEX Keyword)s.)
 
-$(P If present, the $(I ModuleDeclaration) appears syntactically first in the
-source file, and there can be only one per source file.)
+$(P If present, the $(I ModuleDeclaration) must be the first and only such declaration
+in the source file, and may be preceded only by comments.)
```

Comment:
This isn't technically true. `#line` directives may appear before the module declaration.